src/back/index.js

```diff
@@ -2,21 +2,25 @@ const express = require("express");
 require("dotenv").config()
 const cors = require("cors");
 
-const PORT = process.env.PORT;
-const CONN_STR = process.env.CONN_STR;
+const PORT = 8085;
+const CONN_STR = "mongodb://mongodb:27017";
```

Comment:
Is it possible to use the existing environment variables to pass in these hardcoded values? If we can help it, I think it would be best to minimize the changes to this file.

Adding a new route to this file seems easy to port if there are future merge conflicts imo.